setup.cfg

```diff
@@ -1,5 +1,69 @@
-[bdist_wheel]
-universal=1
+[metadata]
+name = environs
+version = 9.5.0
```

Comment:
this would require bumping the version both here and in `__init__.py`, and i'd rather not have two sources of truth.

i suppose we could remove `__version__` in `__init__.py` in the next breaking version. thoughts?

Reply:
Indeed I overlooked this. It's totally possible to keep the previous behavior, I will come back with a change that restore this.